### PR TITLE
Fix typo in set-up-for-high-availability.md

### PR DESCRIPTION
**Pull request by mdsgrafana**
fixed a minor typo

**What this PR does / why we need it**:
There was a typo in the documentation.

**Which issue(s) this PR fixes**:
No defect

Fixes #

**Special notes for your reviewer**:

Changed 
`We recommend finding an expert on for the database you’re using.`
to 
`We recommend finding an expert for the database you’re using.`

### CODE REVIEW

**Comment by achatterjee-grafana** [comment]
Hello @mdsgrafana, the usage of "on" is not incorrect in this line.

**Comment by mdsgrafana** [comment]
> Hello @mdsgrafana, the usage of "on" is not incorrect in this line. Let's keep it as is. Thanks.

The usage of "on for" is incorrect. Either "on" or "for" would be correct but not both of them.

File: docs/sources/administration/set-up-for-high-availability.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ and other persistent data. So the default embedded SQLite database will not work
 
 First, you need to set up MySQL or Postgres on another server and configure Grafana to use that database.
 You can find the configuration for doing that in the [[database]]({{< relref "../administration/configuration.md#database" >}}) section in the Grafana config.
-Grafana will now persist all long term data in the database. How to configure the database for high availability is out of scope for this guide. We recommend finding an expert on for the database you're using.
+Grafana will now persist all long term data in the database. How to configure the database for high availability is out of scope for this guide. We recommend finding an expert for the database you're using.
```
Comment (achatterjee-grafana) [inline docs/sources/administration/set-up-for-high-availability.md:21]:
```suggestion
Grafana will now persist all long term data in the database. How to configure the database for high availability is out of scope for this guide. We recommend finding an expert on the database you're using.
```

**Review by achatterjee-grafana — COMMENTED** [review]
Thanks for the correction. You are correct, on and for does not belong together. Expert on is the correct usage. I was hasty in my earlier review.

File: docs/sources/administration/set-up-for-high-availability.md (hunk 1 above)
Comment (achatterjee-grafana) [reply docs/sources/administration/set-up-for-high-availability.md:21]: Added backport label.

**Review by achatterjee-grafana — CHANGES_REQUESTED** [review]
WIll approve once you are ok with the suggested change. Also added backport label to the PR.

**Pull request merged**